articles/security-center/tutorial-security-incident.md

```diff
@@ -88,8 +88,8 @@ After investigating an alert and understanding its scope, you can respond to sec
 
     This removes the alert from the main alerts list. You can use the filter from the alerts list page to view all alerts with **Dismissed** status.
 
-1.	Optionally, provide feedback on an alert to Microsoft:
-    1. Marking the alert as **Useful** or **Not useful** and providing
+1.	We encoureging you to provide feedback on the alert to Microsoft:
```

Comment:
I really tried to tone this down so that it wasn't a "hard sell". And I added a tip explaining the benefit. 

But if you want to make this addition, please correct the spelling from:
 1.	We encoureging you to provide feedback on the alert to Microsoft:
to
 1.	We encourage you to provide feedback about the alert to Microsoft, marking ......